package.json

```diff
@@ -25,7 +25,8 @@
     "commitlint": "commitlint --from=master",
     "build:client": "rimraf ./client/* && babel client-src/ --out-dir client/ --ignore \"client-src/webpack.config.js\" --ignore \"client-src/modules\" && webpack --config client-src/webpack.config.js",
     "build:types": "rimraf ./types/* && tsc --declaration --emitDeclarationOnly --outDir types && node ./scripts/extend-webpack-types.js && prettier \"types/**/*.ts\" --write && prettier \"types/**/*.ts\" --write",
-    "build": "npm-run-all -p \"build:**\"",
+    "build": "npm run build:client",
+    "postbuild": "npm run-all -p \"build:**\"",
```

Comment:
Better to keep this as it, there are bugs in yarn with this, just improve CI file

Reply:
`npm link webpack-dev-server` is always running `npm run build`, and `npm run build:types` is failing with `webpack-4`

Reply:
I understand this, we should search workarounds, we can try `--ignore-scripts`

Reply:
I am searching for workarounds, We already have `--ignore-scripts` it only ignores `pre` and `post` scripts so it will ignore `prebuild` and `postbuild` scripts only.

Reply:
In theory we create scripts which remove command from `package.json` and run it on CI